app/queues/common/expiration-request.server.ts

```diff
@@ -0,0 +1,105 @@
+import { Worker, Queue, UnrecoverableError } from 'bullmq';
+
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { prisma } from '~/db.server';
+
+import { addNotification } from '../notifications/notifications.server';
+import { deleteDnsRequest } from '../dns/dns-flow.server';
+import { RecordType as RecType } from '../notifications/expiration-notification.server';
+
+import type { ExpirationStatusPayload as ExpirationRequestPayload } from '../notifications/expiration-notification.server';
+
+const { EXPIRATION_JOB_REPEAT_FREQUENCY, JOB_REMOVAL_FREQUENCY } = process.env;
```

Comment:
Set defaults on these so we don't have to provide in env if we don't want to override.